sql-plugin/src/main/scala/com/nvidia/spark/rapids/TypeChecks.scala

```diff
@@ -1624,3 +1631,70 @@ object SupportedOpsDocs {
     }
   }
 }
+
+object SupportedOpsForTools {
```

Comment:
@revans2 would be great if you could look at this part since you did Type checking stuff